eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -622,9 +622,18 @@
   </Match>
 
   <!-- Initial implementation of Storage Blob Batch is using an inline class -->
+  <!-- Storage blob is using an inline class for accessing property -->
   <Match>
-    <Class name="com.azure.storage.blob.batch.BlobBatchAsyncClient"/>
-    <Bug pattern="SIC_INNER_SHOULD_BE_STATIC_ANON"/>
+    <Or>
+      <Class name="com.azure.storage.blob.batch.BlobBatchAsyncClient"/>
+      <Class name="com.azure.storage.blob.BlobContainerAsyncClient"/>
+      <Class name="com.azure.storage.blob.BlobContainerClient"/>
+      <Class name="com.azure.storage.blob.BlobServiceAsyncClient"/>
+      <Class name="com.azure.storage.blob.BlobServiceClient"/>
+      <Class name="com.azure.storage.blob.specialized.BlobAsyncClientBase"/>
+      <Class name="com.azure.storage.blob.specialized.BlobClientBase"/>
+      <Bug pattern="SIC_INNER_SHOULD_BE_STATIC_ANON"/>
```

Comment:
Is there a problem with making these private static classes?

Reply:
Not really. Will have private class